scripts/test-runner.js

```diff
@@ -188,14 +187,15 @@ if (process.argv.length === 3 || process.argv.length === 4) {
         testType = 'integration';
     } else if (process.argv[2] === 'all') {
         if (process.argv.length === 4) {
-            testCommand = `node node_modules/mocha/bin/mocha -g "${process.argv[3]}" ` +
+            testCommand = `node node_modules/mocha/bin/mocha -g "${process.argv[3]}" "test/unit/**/*.js" && `;
+            testCommand += `node node_modules/mocha/bin/mocha -g "${process.argv[3]}" ` +
                 '"test/integration/**/serial/**/*.js" && ';
-
             testCommand += `node node_modules/mocha/bin/mocha -j 8 --parallel -g  "${process.argv[3]}" ` +
-                '"test/**/+(parallel|unit)/**/*.js"';
+                '"test/**/parallel/**/*.js"';
         } else {
-            testCommand = 'node node_modules/mocha/bin/mocha "test/**/serial/**/*.js" && ';
-            testCommand += 'node node_modules/mocha/bin/mocha -j 8 --parallel "test/**/+(parallel|unit)/**/*.js"';
+            testCommand = 'node node_modules/mocha/bin/mocha "test/unit/**/*.js" && ';
+            testCommand += 'node node_modules/mocha/bin/mocha "test/**/serial/**/*.js" && ';
+            testCommand += 'node node_modules/mocha/bin/mocha -j 8 --parallel "test/**/parallel/**/*.js"';
```

Comment:
Reason of change: TranslateAddressProviderTest runs a socket server at 5701 in the unit test. That test failed due to running in parallel with parallel tests. I think we can separate unit tests from parallel tests and run them separately. Unit tests only takes 32 seconds.